File: src/es6TemplateProvider.java
import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider;
import org.jetbrains.annotations.Nullable;

public class Es6TemplateProvider implements DefaultLiveTemplatesProvider {
    @Override
    public String[] getDefaultLiveTemplateFiles() {
        return new String[]{"liveTemplates/es6Intentions"};
    }

    @Nullable
    @Override
    public String[] getHiddenLiveTemplateFiles() {
        return new String[0];
    }
}

File: src/Es6TemplateProvider.kt
import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider

class Es6TemplateProvider : DefaultLiveTemplatesProvider {
    override fun getDefaultLiveTemplateFiles(): Array<String> {
        return arrayOf("liveTemplates/es6Intentions")
    }

    override fun getHiddenLiveTemplateFiles(): Array<String?> {
        return arrayOfNulls(0)
    }
}